sdk/highlight-next/src/util/with-highlight-edge.ts

```diff
@@ -19,7 +19,7 @@ export function Highlight(env: HighlightEnv) {
 		) {
 			H.initEdge(request, env, event)
 			const headers: IncomingHttpHeaders = {}
-			request.headers.forEach((k, v) => (headers[k] = v))
+			request.headers.forEach((k, v) => (headers[v] = k))
```

Comment:
This was baffling. IT took me a while to realize that the keys and values were flipped!

Reply:
See https://developer.mozilla.org/en-US/docs/Web/API/Headers/forEach